calendar/js/script1.js

```diff
@@ -0,0 +1,237 @@
+     new Birthday ("Darshan", new Date(1997,0,15)),
+     new Birthday ("Gurpreet Chhabra", new Date(1995,3,2)),
+     new Birthday ("Sonam Ravi Gupta", new Date(1987,3,22)),
+     new Birthday ("Siyaram Patidar", new Date(1985,4,3)),
+     new Birthday ("Shubham Choubey", new Date(1993,4,8)),
+     new Birthday ("Mayur Vaidya", new Date(1994,4,9)),
+     new Birthday ("Amit Nagar", new Date(1986,4,10)),
+     new Birthday ("Deepak Patidar", new Date(1990,4,10)),
+     new Birthday ("Rahul Kulmi", new Date(1988,4,28)),
+     new Birthday ("Vishal Patidar", new Date(1994,5,20)),
+     new Birthday ("Awanish Tiwari", new Date(1974,6,6)),
+     new Birthday ("Surendra Patidar", new Date(1988,6,21)),
+     new Birthday ("Anjana Singh", new Date(1992,6,24)),
+     new Birthday ("Aditiya Paliwal", new Date(1994,7,8)),
+     new Birthday ("Varsha Tyagi", new Date(1992,9,13)),
+     new Birthday ("Rashmi Soni", new Date(1993,9,19)),
+     new Birthday ("Priyanshi Asawara", new Date(1993,10,19)),
+     new Birthday ("Shashank Saxena", new Date(1990,11,11)),
+     new Birthday ("Nitesh Thakhur", new Date(1990,11,12)),
+     new Birthday ("Satya Naryan Patidar", new Date(1983,11,12)),
+ ];
+
+var birth = {
+    "January" : [{"name" : "Shubham Katariya" , "date" : "3-1-1995"},
+                 {"name" : "Vipin Joshi" , "date" : "15-1-1946"},
+                 {"name" : "Shikha Shakarwar" , "date" : "15-1-1994"},
+                 {"name" : "Darshana" , "date" : "15-1-1997"} ],
+    "Febraury" : [],
+    "March" : [],
+    "April" : [{"name" : "Gurpreet Chhabra", "date" : "2-4-1995"},
+               {"name" : "Sonam Ravi Gupta", "date" : "22-4-1987"} ],
+    "May" : [{"name" : "Siyaram Patidar", "date" : "3-5-1985"},
+             {"name" : "Shubham Choubey", "date" : "9-5-1993"},
+             {"name" : "Mayur Vaidya", "date" : "9-5-1994"},
+             {"name" : "Amit Nagar", "date" : "10-5-1986"},
+             {"name" : "Deepak Patidar", "date" : "10-5-1990"},
+             {"name" : "Rahul Kulmi", "date" : "28-5-1988"} ],
+    "June" : [{"name" : "Vishal Patidar", "date" : "20-6-1994"} ],
+    "July" : [ {"name" : "Awanish Tiwari", "date" : "6-7-1974"},
+               {"name" : "Surendra Patidar", "date" : "21-7-1988"},
+               {"name" : "Anjana Singh", "date" : "24-7-1992"} ],
+    "August" : [{"name" : "Aditiya Paliwal", "date" : "8-7-1994"} ],
+    "September" : [{"name" : "Varsha Tyagi", "date" : "13-9-1992"} ,
+                   {"name" : "Rashmi Soni", "date" : "19-9-1993"},
+                   {"name" : "Rupak", "date" : "22-9-1995"} ],
+    "October" : [{"name" : "Priyanshi Asawara", "date" : "19-10-1993"} ,
+                 {"name" : "Piyush ", "date" : "16-10-1996"} ],
+    "November" : [],
+    "December" : [{"name" : "Shashank Saxena", "date" : "11-12-1990"} ,
+                  {"name" : "Nitesh Thakhur", "date" : "12-12-1990"} ,
+                  {"name" : "Satya Naryan Patidar", "date" : "12-12-1983"} ]
+};
```

Comment:
code is not intent for `birth` json and also rename `birth` var name as it is sounds like incomplete